docs/topics/forms/descriptions/location-description.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Description location
+layout: default
+parent: Input and description
+nav_order: 4
+---
+
+# Place descriptions between label and form field
+
+{: .info }
+Note: this is best practice and not a WCAG requirement.
+
+Users read from the top down in a form. Place descriptions in a consistent location, preferably between the label and the text form field. Then information appears after the label at a logical point in the reading order.
+
+This also reduces the chance of the information overlapping with, for example, browser popups. For more information and user research, read the article [Avoid Messages Under Fields](https://adrianroselli.com/2017/01/avoid-messages-under-fields.html) by Adrian Roselli.
```

Comment:
Maybe: "This also reduces the chance of the information overlapping with browser and browser toolbar popups."

Removes subclause, expands example.